### PR TITLE
Update pd-configuration-file.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

- Update configuration values
- Add two more configuration items

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3238
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @CaitinChen

File: pd-configuration-file.md [hunk 1]
```diff
@@ -211,7 +211,7 @@ Configuration items related to scheduling
 ### `tolerant-size-ratio`
 
 + Controls the `balance` buffer size
-+ Default value: `5`
++ Default value: `0` (automatically adjusted buffer size)
```
Comment (CaitinChen) [inline pd-configuration-file.md:214]:
```suggestion
+ Default value: `0` (automatically adjusts the buffer size)
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -256,7 +256,20 @@ Configuration items related to replicas
 ### `location-labels`
 
 + The topology information of a TiKV cluster
-+ Default value: `[]`
++ Default value: []
++ [Cluster topology configuration](/location-awareness.md)
+
+### `strictly-match-label`
+
++ Enables the strict check for whether the TiKV label matches PD's location-label.
```
Comment (CaitinChen) [inline pd-configuration-file.md:264]:
```suggestion
+ Enables the strict check for whether the TiKV label matches PD's `location-labels`.
```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -256,7 +256,20 @@ Configuration items related to replicas
...
++ Default value: `false`
+
+### `enable-placement-rules`
+
++ Enables `placement-rules`.
++ Default value: `false`
++ See [Placement Rules](/configure-placement-rules.md).
++ An experimental feature of v4.0.
```
Comment (CaitinChen) [inline pd-configuration-file.md:272]:
```suggestion
+ An experimental feature of TiDB 4.0.
```

File: pd-configuration-file.md [hunk 4]
```diff
@@ -256,7 +256,20 @@ Configuration items related to replicas
 ### `location-labels`
 
 + The topology information of a TiKV cluster
-+ Default value: `[]`
++ Default value: []
```
Comment (lilin90) [inline pd-configuration-file.md:259]:
```suggestion
+ Default value: `[]`
```

**Pull request merged**